financialaid/views.py

```diff
@@ -234,3 +223,52 @@ class FinancialAidActionView(UpdateAPIView):
     lookup_field = "id"
     lookup_url_kwarg = "financial_aid_id"
     queryset = FinancialAid.objects.all()
+
+
+class GetLearnerPriceForCourseView(APIView):
+    """
+    View for retrieving a leaner's price for a course run
+    """
+    authentication_classes = (SessionAuthentication, )
+    permission_classes = (IsAuthenticated, )
+
+    def get(self, request, *args, **kwargs):
+        """
+        Get request for GetLearnerPriceForCourseView
```

Comment:
Can you create a function `get_price_for_user(user, program)` in `api.py` and move some of the functionality here into that function? I think it would be useful for ecommerce work in the near future.